### PR TITLE
CHE-864 Adapt Subversion project importer UI to 4.x

### PR DESCRIPTION
**Pull request by stour**
Signed-off-by: Stéphane Tournié stephane.tournie@serli.com

@skabashnyuk @vparfonov

### CODE REVIEW

File: plugins/plugin-svn/che-plugin-svn-ext-ide/src/main/java/org/eclipse/che/plugin/svn/ide/importer/SubversionProjectImporterPresenter.java [hunk 1]
```diff
@@ -29,63 +33,104 @@
 public class SubversionProjectImporterPresenter extends AbstractWizardPage<ProjectConfigDto>
         implements SubversionProjectImporterView.ActionDelegate {
 
-    private SubversionProjectImporterView view;
+    // start with white space
+    private static final RegExp SUBVERSION_REPOSITORY_REGEX =
+            RegExp.compile("^(http|https|svn|svn\\+ssh)://[A-Za-z0-9_\\-]+(\\.[A-Za-z0-9_\\-:]+)+(/[A-Za-z0-9_.\\-]+)*/?$");
```
Comment (skabashnyuk) [inline plugins/plugin-svn/che-plugin-svn-ext-ide/src/main/java/org/eclipse/che/plugin/svn/ide/importer/SubversionProjectImporterPresenter.java:38]: Do we have a unit  tests for this regexp?

Comment (stour) [inline plugins/plugin-svn/che-plugin-svn-ext-ide/src/main/java/org/eclipse/che/plugin/svn/ide/importer/SubversionProjectImporterPresenter.java:38]: It's tested with both wrong and correct values in [SubversionProjectImporterTest](https://github.com/stour/che/blob/CHE-864/plugins/plugin-svn/che-plugin-svn-ext-ide/src/test/java/org/eclipse/che/plugin/svn/ide/importer/SubversionProjectImporterPresenterTest.java#L127-L163) yes.

**Comment by skabashnyuk** [comment]
ok for me. @vparfonov ?

**Comment by vparfonov** [comment]
ok

**Comment by stour** [comment]
thanks

**Pull request merged**